cmake/FindCairo.cmake

```diff
@@ -38,14 +38,14 @@ FIND_PATH(CAIRO_INCLUDE_DIR
     PATH_SUFFIXES cairo
 )
 
-IF(APPLE)
+IF(APPLE OR CMAKE_SYSTEM_NAME MATCHES "OpenBSD")
```

Comment:
I would completely remove the IF() test to always execute this block. It is not at all OS specific

Reply:
sure, force-pushed a removal of the IF, makes more sense